### PR TITLE
fix(argo-cd): Set TTL on redis-secret-init Job

### PR DESCRIPTION
**Pull request by jkleinlercher**
set `ttlSecondsAfterFinished` to prevent stuck during sync when managed by argocd

should fix https://github.com/argoproj/argo-cd/issues/6880#issuecomment-2263802072

---
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by tico24** [comment]
Thanks for your PR. Please address the title formatting, the chart version and changelog (essentially the tick boxes in the description of your PR). I have moved this PR back to draft while you work on it.

**Comment by jkleinlercher** [comment]
> Thanks for your PR. Please address the title formatting, the chart version and changelog (essentially the tick boxes in the description of your PR). I have moved this PR back to draft while you work on it.

@tico24 I think this PR is ready for a review

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -26,5 +26,5 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: changed
-      description: Bump argo-cd to v2.11.4
+    - kind: added
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:29]:
```suggestion
    - kind: fixed
```

Ref:
> supported kinds are added, changed, deprecated, removed, fixed and security.
> https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations

**Review by tico24 — APPROVED** [review]

**Comment by venkatamutyala** [comment]
@oponomarov-tu i recommend opening a new github issue and just referencing this one

**Pull request merged**